schema/proto3/common.proto

```diff
@@ -34,10 +35,11 @@ enum OperationType {
     GET = 2;
     DELETE = 3;
     INFO = 4;
-    FINALIZE = 5;
-    CREATE_UPDATE_SWITCH = 6;
-    CREATE_UPDATE_ROUTER = 7;
-    CREATE_UPDATE_GATEWAY = 8;
+    NEIGHBOR_INFO = 5; // FIXME: we may need CRUD for NEIGHBOR_INFO
```

Comment:
Let us close on all the FIXME staff and plan to merge it :-)

Reply:
Fixed.